runtime/lua/vim/lsp.lua

```diff
@@ -402,27 +402,36 @@ do
       if state.use_incremental_sync then
         -- This must be done immediately and cannot be delayed
         -- The contents would further change and startline/endline may no longer fit
-        table.insert(state.pending_changes, incremental_changes(client))
+        if not state.pending_changes[uri] then
```

Comment:
Need to make sure URI is sanitized on windows. We can also track per bufnr like elsewhere but URI is more clear in this case IMO.

Reply:
No, changedtick is used for the `version` parameter and is buffer-local

Reply:
Also when debounce is off didChange is effectively sent per-buffer and not per-URI

Reply:
Can you also update the comment here: 

https://github.com/neovim/neovim/blob/b5b025f6a3ad406929c096f63b5f1e7649d3c818/runtime/lua/vim/lsp.lua#L323

Reply:
> No, changedtick is used for the version parameter and is buffer-local

This isn't relevant, because URI is effectively bufnr in this case.

> Also when debounce is off didChange is effectively sent per-buffer and not per-URI

I don't see the issue with this

Reply:
What's the deal with URI sanitization on Windows, then?